src/css/theme/essentials-base16.css

```diff
@@ -0,0 +1,6 @@
+@theme {
+  --text-root: 100%;
+  --text-input: 1.125rem;
+  --font-size: 0.0625rem;
+  --spacing: 0.0625rem;
+}
```

Comment:
This is the trick - updating this scale factor I use for the text-__ and spacing (eg, py-___) utilties